### PR TITLE
docs: update URLs and contact addresses in man page

### PR DESCRIPTION
**Pull request by masatake**
We have to update the URLs and contact addresses in our documents so we can receive bug ports.
This is the first step.

### CODE REVIEW

File: Lsof.8 [hunk 1]
```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, lsof-org at GitHUB team maintains.
```
Comment (lrosenman) [inline Lsof.8]: Slightly better English would be:

Since version 4.93.0, the lsof-org team at GitHub is maintaining 
.I Lsof

Comment (masatake) [reply Lsof.8]: Thank you, I will update the change. BTW, cirrus-ci for freebsd13 is failed for this PR.

Comment (lrosenman) [reply Lsof.8]: Looks like a flake on cirrus.  I requested a re-run.  Will check back in a bit.

Comment (lrosenman) [reply Lsof.8]: The re-run was successful (look further down in the list).

Comment (masatake) [reply Lsof.8]: Oh, I see.

File: 00DIST [hunk 2]
```diff
@@ -4999,5 +4999,5 @@ July 14, 2018
                 [FreeBSD] update to include <sys/_lock.h> on recent -CURRENT
 		since it is no longer implicitly included via header pollution.
 
-Masatake YAMATO <yamato@redhat.com>, a member of lsof-org
+Masatake YAMATO <yamato@redhat.com>, a member of lsof-org team at GitHUB
```
Comment (lrosenman) [inline 00DIST]: a member of the lsof-org team at GitHub

File: Lsof.8 [hunk 3]
```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, lsof-org team at GitHUB maintains.
```
Comment (lrosenman) [inline Lsof.8]: still would like to see:

Since version 4.93.0, the lsof-org team at GitHub maintains lsof.

File: README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 [![Coveralls Linux Coverage Status on Travis CI](https://coveralls.io/repos/github/lsof-org/lsof/badge.svg?branch=master)](https://coveralls.io/github/lsof-org/lsof?branch=master)
 
 # lsof
-lsof-org at GitHub team takes over the maintainership of lsof
+lsof-org team at GitHub takes over the maintainership of lsof
```
Comment (lrosenman) [inline README.md]: the lsof-org team at GitHub takes over the maintainership of lsof

**Comment by masatake** [comment]
Thank you. Updated again.

File: Lsof.8 [hunk 5]
```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, the lsof-org team at GitHub maintains.
```
Comment (lrosenman) [inline Lsof.8]: I talked to my wife who is an editor and great person on grammar.  
Ending the sentence with maintains.  makes it a question: (maintains what?).

Can we add lsof between the maintains and the period?

Comment (masatake) [reply Lsof.8]: Oh, I see.

Comment (lrosenman) [reply Lsof.8]: I do understand English grammar is hard, and definitely not your first language. I'm sure I'd make a mess of Japanese.

**Comment by masatake** [comment]
Thank you very much.

**Pull request merged**